src/plugin/main.c

```diff
@@ -200,6 +202,13 @@ static bool reload_button_clicked(obs_properties_t *props, obs_property_t *prope
 	return true;
 }
 
+static void reload_on_scene(void* vptr)
+{
+	struct browser_data* data = vptr;
+	if (data->reload_on_scene)
+		reload_button_clicked(NULL, NULL, vptr);
```

Comment:
Better use `browser_manager_reload_page` here

Reply:
`reload_button_clicked` is a callback specific to button click and might do stuff other than just reload a page.

Reply:
Will be done.